lib/java/logging-service-telemetry/src/main/java/module-info.java

```diff
@@ -0,0 +1,14 @@
+module org.enso.logging.service.telemetry {
+  requires java.net.http;
+  requires scala.library;
```

Comment:
Why this project requires Scala?
- it seems to be fully written in Java
- logback API is also written in Java
- there shall be no need for Scala around

Reply:
See https://github.com/enso-org/enso/pull/12600#discussion_r2008677051